packages/node_modules/@webex/plugin-meetings/test/unit/spec/members/request.js

```diff
@@ -118,5 +118,25 @@ describe('plugin-meetings', () => {
         assert.equal(requestParams.body.hand.raised, true);
       });
     });
+
+    describe('#lowerAllHands', () => {
```

Comment:
issue: There needs to be tests here for the error cases:

- Missing `options`
- Missing `locusUrl`
- Missing `requestingParticipantId`